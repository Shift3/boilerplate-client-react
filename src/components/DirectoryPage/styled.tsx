import styled from 'styled-components';

export const DirectoryPageContainer = styled.div`
  min-height: ${(props) => props.theme.minPageHeight};
`;