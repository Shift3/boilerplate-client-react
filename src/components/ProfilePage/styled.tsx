import styled from 'styled-components';

export const ProfilePageContainer = styled.div`
  min-height: ${(props) => props.theme.minPageHeight};
`;