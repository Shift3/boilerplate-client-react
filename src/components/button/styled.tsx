import styled from 'styled-components';
import { Button } from 'react-bootstrap';
import StyleValues from '../../utils/styleValues';

const { accent, accentDisabled } = StyleValues;

export const CustomButton = styled(Button)`
  background-color: ${(props) => props.theme.backgroundColor};
  border-color: ${(props) => props.theme.borderColor};
  color: ${(props) => props.theme.color};
  width: ${(props) => props.theme.width};
  &:hover {
    background-color: ${(props) => props.theme.hoverBackgroundColor};
    border-color: ${(props) => props.theme.hoverBorderColor};
  }
  &:disabled {
    background-color: ${(props) => props.theme.disabledBackgroundColor};
    border-color: ${(props) => props.theme.disabledBorderColor};
  }
`;

CustomButton.defaultProps = {
  theme: {
    backgroundColor: accent,
    borderColor: accent,
    color: '#fff',
    disabledBackgroundColor: accentDisabled,
    disabledBorderColor: accent,
    hoverBackgroundColor: accent,
    hoverBorderColor: accent,
    width: '90%'
  }
};