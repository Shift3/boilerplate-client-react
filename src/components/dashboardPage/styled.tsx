import styled from 'styled-components';

export const DashboardPageContainer = styled.div``;