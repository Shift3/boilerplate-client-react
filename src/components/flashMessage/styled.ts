import styled from 'styled-components';

export const FlashMessageContainer = styled.div``;
