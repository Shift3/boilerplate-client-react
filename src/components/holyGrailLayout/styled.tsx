import styled from 'styled-components';
import { } from '../../utils/styleValues';

export const HolyGrailWrapper = styled.div`
  display: grid;
  grid-template-columns: repeat(6, 1fr);
    grid-auto-rows: minmax(100px, auto);
    grid-template-areas:
      "hd   hd   hd   hd   hd  hd"
      "lsd main main main main rsd"
      "ft   ft   ft   ft   ft  ft";
`;

export const HolyGrailHeader = styled.div`
    grid-area: hd;
`;

export const HolyGrailLeftSideBar = styled.div`
  grid-area: lsd
  `;

export const HolyGrailMain = styled.div`
  grid-area: main
  `;

export const HolyGrailRightSideBar = styled.div`
  grid-area: rsd
  `;

export const HolyGrailFooter = styled.div`
  grid-area: ft
  `;
