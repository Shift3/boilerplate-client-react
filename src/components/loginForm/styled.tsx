import Button from 'react-bootstrap/Button';
import Form from 'react-bootstrap/Form';
import styled from 'styled-components';

export const StyledForm = styled(Form)`
  padding: 60px;
  border-radius: 5px;
  min-width: 450px;
`;

export const Title = styled.div`
  color: ${(props) => props.theme.cardHeader};
  font-size: 2em;
  font-style: bold;
  padding-bottom: 10px;
`;

export const FormLabel = styled(Form.Label)`
  color: white;
`;

export const InputError = styled.span`
  color: lightgrey;
`;

export const ButtonWrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  padding-top: 10px;
`;

export const CancelButton = styled(Button)`
  color: white;
  background-color: ${(props) => props.theme.primary};
  border: 1px solid #679daa;
  margin-right: 20px;
`;

export const LogInButton = styled(Button)`
  color: white;
  background-color: ${(props) => props.theme.accent};
  border: none;
`;
