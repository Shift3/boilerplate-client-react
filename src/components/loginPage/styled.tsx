import styled from 'styled-components';
import { Button } from 'react-bootstrap';

export const Wrapper = styled.div`
  height: 100%;
  width: 100%;
  background-color: ${(props) => props.theme.authBackground};
  padding: 40px;
  display: flex;
  justify-content: center;
  align-items: center;
`;

export const FormLeft = styled.div``;

export const FormRight = styled.div``;

export const CreateAccountButton = styled(Button)``;
