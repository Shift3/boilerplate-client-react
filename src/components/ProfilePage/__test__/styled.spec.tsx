import { expectToMatchSnapshot } from '../../../utils/test';
import { ProfilePageContainer } from '../styled';

import 'jest-styled-components';

describe('<ProfilePage /> styled component snapshot tests', () => {
  it('Should match the stored <ProfilePageContainer /> snapshot', () =>
    expectToMatchSnapshot(<ProfilePageContainer />));
});