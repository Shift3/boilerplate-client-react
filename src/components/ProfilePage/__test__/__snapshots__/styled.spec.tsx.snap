// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfilePage /> styled component snapshot tests Should match the stored <ProfilePageContainer /> snapshot 1`] = `
<div
  className=""
/>
`;
